compiler/src/typed/typed_well_formedness.re

```diff
@@ -228,9 +245,18 @@ module WellFormednessArg: TypedtreeIter.IteratorArgument = {
         )
           when func == "==" || func == "!=" =>
         if (List.exists(((_, arg)) => exp_is_wasm_unsafe(arg), args)) {
+          let (_, arg) = List.nth(args, 0);
```

Comment:
```suggestion
          let (_, arg) = List.hd(args);
```
this should either use `List.hd` or a pattern match. Note that both `nth` and `hd` raise on an empty list. How are we certain this list contains an element?

Reply:
I went with the pattern match, and have a fallback where we show all the possible modules because we dont know the module in that case. It is worth noting I dont think that we can hit here without having args as the typechecker will throw a warning that you did not include enough arguments.